cli/emit.rs

```diff
@@ -100,17 +100,9 @@ impl Emitter {
         source.clone(),
         media_type,
       )?;
-      let transpiled_source = match parsed_source
-        .transpile_owned(&self.transpile_options, &self.emit_options)
-      {
-        Ok(result) => result?,
-        Err(parsed_source) => {
-          // transpile_owned is more efficient and should be preferred
-          debug_assert!(false, "Transpile owned failed.");
-          parsed_source
-            .transpile(&self.transpile_options, &self.emit_options)?
-        }
-      };
+      let transpiled_source = parsed_source
+        .transpile(&self.transpile_options, &self.emit_options)?
+        .into_source();
```

Comment:
There is a variant on this return value that says whether it had to clone or whether it was owned. Can you revert the code more back to the original and use that new value? We want to ensure we're not cloning when emitting here.

Reply:
Done in #23421